internal/gatewayapi/route.go

```diff
@@ -553,6 +562,28 @@ func (t *Translator) ProcessTLSRoutes(tlsRoutes []*v1alpha2.TLSRoute, gateways [
 	return relevantTLSRoutes
 }
 
+func getUpstreamConfig(obj client.Object) ir.UpstreamConfig {
+	var (
+		enableProxyProtocol bool
+		maxConns            uint32
+	)
+
+	annotations := obj.GetAnnotations()
+	_, enableProxyProtocol = annotations[AnnotationUpstreamProxyProtocol]
```

Comment:
I would make this opt-in via a specific value (i.e., `true`). Empty annotations probably shouldn't change the behavior.

Reply:
Using case-insensitive true [2455353](https://github.com/gravitational/gateway/pull/4/commits/245535374e5be2e0b3697a8e3bdc1c33fb883ebe)